app/src/store/cart.ts

```diff
@@ -324,13 +421,48 @@ export default function useCartStore() {
     return summary.slice(0, -3); // trim the trailing ` + ` from the string
   });
 
+  /**
+   * @notice Returns all clr matching for each grant and each round
+   */
+  const clrPredictions = computed<CartPredictions>(() => {
+    const _predictions: CartPredictions = {};
+    cart.value.forEach((item) => {
+      _predictions[item.grantId] = getPredictionForAmount(
+        item.grantId,
+        item.contributionAmount,
+        item.contributionToken
+      ) as CartPrediction[];
+    });
+
+    return _predictions;
+  });
+
+  const clrPredictionsByToken = computed<{ [key: string]: number }>(() => {
```

Comment:
Can you add a comment describing this property? I think it takes `clrPredictions` and groups/sums the match results by token, similar to what `cartSummary` does to `cart`, but want to confirm